src/vsr/superblock.zig

```diff
@@ -640,6 +650,8 @@ pub fn SuperBlockType(comptime Storage: type) type {
                     .commit_max = 0,
                     .log_view = 0,
                     .view = 0,
+                    .replica = options.replica,
+                    .replica_count = options.replica_count,
```

Comment:
These should be zeroes — this superblock header is only used to generate the root checksum, it is never actually written to disk. It isn't expected to be a valid superblock (e.g. `vsr_headers_count == 0`).

Reply:
I tried that, and it didn't work, as it trips asserts that these don't change. Note that the replica was non-zero before the change here.

I think I'll make these zero once configurationn is actually dynamic.